src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -52,8 +53,23 @@ x-ref-data:
           name: description
           type: *string
           remarks: A longer (optional) plain-language description of this entity, for use in wider list views.
+      - column: &session_timestamp_column
+          type: *session_timestamp
+          constraints:
+            nullable: false
 
 databaseChangeLog:
+  - property:
+      name: attribute_bytes_type
+      value: BYTEA
+      dbms: postgresql
+  - property:
```

Comment:
Arguably this could be deleted, since we no longer support hsqldb, but then the fallback is also unnecessary if you want to get technical.